packages/components/pie-checkbox-group/src/index.ts

```diff
@@ -75,18 +80,25 @@ export class PieCheckboxGroup extends FormControlMixin(RtlMixin(LitElement)) imp
         const {
             name,
             label,
+            isInline,
             assistiveText,
             status,
             disabled,
         } = this;
+
+        const dynamicClasses = {
+            'c-checkboxGroup--inline': isInline,
+        };
```

Comment:
This is an alternative which keeps all the definitions together, what do you think? (Of course, L99 would also need updating) I think Ash does something similar in his classMap PR
```suggestion
        const classes = {
            'c-checkboxGroup': true
            'c-checkboxGroup--inline': isInline,
        };
```